storage/scripts/poc.sh

```diff
@@ -31,13 +31,20 @@ tests_poc() {
         fi
     done
     curl --fail --insecure --user spdkuser:spdkpass -X POST -H 'Content-Type: application/json' -d '{"id": 1, "method": "bdev_get_bdevs"}' http://127.0.0.1:9009
-    docker-compose run opi-spdk-client
```

Comment:
for me this is avoiding the problem and not fixing it... once yo did compose up, you should be able to run the client as many times as you want... client configures servers by sending apis, you should be able to repeat that process...